addons/web_editor/static/lib/odoo-editor/test/utils.js

```diff
@@ -10,6 +10,9 @@ export const Direction = {
     FORWARD: 'FORWARD',
 };
 
+let thisTestIsMobile = false;
+let nextTestIsMobile = false;
```

Comment:
I think those variable names are confusing because it is not really the "next" test, it is in fact the _same_ test, just run differently. From the perspective of mocha, it is even a single test containing both run.

I think I'd name those variables `isMobile` and `hasMobile` respectively but I guess `hasMobile` sounds a bit vague and weird... Maybe `isMobileTest` and `hasMobileTest` ? What do you think ?

Reply:
I don't like any of the names I've thought of so far. Your last suggestion works for me so let's go with that.

Reply:
Added some basic comments to the variable declarations. Let me know what you think @dmo-odoo